### PR TITLE
image-index: define platform.os.features on Windows

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Please refer to #632 for previous discussion.

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

File: image-index.md [hunk 1]
```diff
@@ -59,7 +59,10 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.features`** *array of strings*
 
-        This OPTIONAL property specifies an array of strings, each specifying a mandatory OS feature (for example on Windows `win32k`).
+        This OPTIONAL property specifies an array of strings, each specifying a mandatory OS feature.
+        For Windows, image indexes SHOULD use, and implementations SHOULD understand the following values:
+		- `win32k`: image requires `win32k.sys` on the host (Note: `win32k.sys` is missing on Nano Server)
+        Convention for other features SHOULD be submitted to this specification for standardization.
```
Comment (wking) [inline image-index.md]: I liked your #650 wording for this better.  How about:

> When `os` is not `windows`, values are implementation-defined and SHOULD be submitted to this specification for standardization.

Comment (AkihiroSuda) [inline image-index.md]: done

**Comment by vbatts** [comment]
@RobDolinMS can you get someone to review this, please?

**Comment by AkihiroSuda** [comment]
@RobDolinMS PTAL?

**Comment by RobDolinMS** [comment]
@darrenstahlmsft would you please give this a look?

**Comment by RobDolinMS** [comment]
Tagging @jstarks to take a look.

**Comment by vbatts** [comment]
bump

**Comment by RobDolinMS** [comment]
@jstarks has been out-of-office but he should be back soon and has been asked to review.

**Comment by q384566678** [comment]
Waiting for results.

**Comment by vbatts** [comment]
we're 30+ days on getting a review for this. I'm in favor of bumping this to post 1.0 so it is not a blocker

**Comment by philips** [comment]
100%, lets move to post 1.0

**Comment by stevvooe** [comment]
I move to open another PR to simply reserve these fields.

@AkihiroSuda Do you mind leaving this PR open and we'll reserve these fields for a future version?

**Comment by AkihiroSuda** [comment]
no problem

**Comment by vbatts** [comment]
@stevvooe so, post-v1.0.0? or are you opening a PR for the reserve language?

**Comment by jstarks** [comment]
Maybe this is a day late and a dollar short, but LGTM.

**Comment by vbatts** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**